src/PKSim.Core/Events/ProgressDoneWithMessageEvent.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PKSim.Core.Events
+{
+   public class ProgressDoneWithMessageEvent
```

Comment:
Why do we need this eventt in PKsim? don't we have something generic in core for this?

Reply:
Considerign there is EXACTLY the same class in Core and that a ProgresMessage with Even is not linked to MoBi or PKSim, it should be in Core. 
```
  public class ProgressDoneWithMessageEvent
  {
     public string Message { get; }

     public ProgressDoneWithMessageEvent(string message)
     {
        Message = message;
     }
  }
```

this is the code in MoBi. IMO that belongs in Core